components/snackbar/Snackbar.js

```diff
@@ -19,7 +19,7 @@ const factory = (Button) => {
         PropTypes.element,
       ]),
       onClick: PropTypes.func,
-      onTimeout: PropTypes.func,
+      onTimeout: PropTypes.func, // eslint-disable-line
```

Comment:
Should there be an additional lint directive here to specify exactly what you are trying to disable?

Reply:
Yes, but I have to review the rest of the components, because if we are doing this in a lot more places maybe it's better to disable the rule.

Since we do destructuring in some components to pass props to child components I'm guessing this is happening in more places.